lib/generators/shopify_app/install/install_generator.rb

```diff
@@ -70,8 +72,37 @@ def insert_hosts_into_development_config
         )
       end
 
+      protected
```

Comment:
Why `protected` rather than `private`?

Reply:
Oh good catch, this was leftover from testing. It should be private!